src/AS_02_ACES.h

```diff
@@ -278,24 +278,8 @@ const byte_t TIFFMagicBE[4] = { 0x4d, 0x4d, 0x00, 0x2a };
 //
 int const NS_ID_LENGTH = 16;
 //
-static byte_t s_ns_id_target_frame_prefix[NS_ID_LENGTH] =
-{
-  // RFC 4122 type 5
-  // 2067-50 8.2.5.4 / RFC4122 Appendix C
-  //urn:uuid:bba41561-c505-4c9c-ab5a-71c68c2d70ea
-  0xbb, 0xa4, 0x15, 0x61, 0xc5, 0x05, 0x4c, 0x9c,
-  0xab, 0x5a, 0x71, 0xc6, 0x8c, 0x2d, 0x70, 0xea
-};
-static byte_t  s_asset_id_prefix[NS_ID_LENGTH] =
-{
-  // RFC 4122 type 5
-  // 2067-2:2016 7.3.1
-    0xaf, 0x86, 0xb7, 0xec, 0x4c, 0xdf, 0x4f, 0x9f,
-    0x82, 0x0f, 0x6f, 0xd8, 0xd3, 0x00, 0x30, 0x23
-};
 // Generate UUID asset ID values from target frame file contents
 AS_02::Result_t CreateTargetFrameAssetId(Kumu::UUID& rID, const std::string& target_frame_file);
-static Kumu::UUID create_4122_type5_id(const byte_t* subject_name, Kumu::fsize_t size, const byte_t* ns_id);
```

Comment:
With these changes applied, AS_02_ACES.cpp does not compile anymore. Please leave as is.

Reply:
Sorry, that was a silly mistake!  These bits are back in a2776a07c946d52f17c4f464774b19923d15b215